test/plugins/data/templateTags.json

```diff
@@ -0,0 +1,18 @@
+[{
+    "id": 7969,
+    "name": "screwdriver/build",
+    "tag": "stable",
+    "version": "0.0.1"
+},
+{
+    "id": 7970,
+    "name": "screwdriver/build",
+    "tag": "stable",
```

Comment:
Hmm to be accurate, 1 tag can only be mapped to 1 version. There cannot be multiple versions with the `stable` tag. For example, `stable` can only map to `3.2.1`. 
However, 1 version can have multiple tags. For example, `3.2.1` can be `stable` and `latest` at the same time. It's very similar to docker hub tags.